internal/os/os_test.go

```diff
@@ -0,0 +1,82 @@
+package os
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+
+	root "github.com/equinix/metal-cli/internal/cli"
+	outputPkg "github.com/equinix/metal-cli/internal/outputs"
+	"github.com/spf13/cobra"
+)
+
+func TestCli_OperatingSystem(t *testing.T) {
+	subCommand := "operating-systems"
+	consumerToken := ""
+	apiURL := ""
+	Version := "metal"
+	rootClient := root.NewClient(consumerToken, apiURL, Version)
+	type fields struct {
+		MainCmd  *cobra.Command
+		Outputer outputPkg.Outputer
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		want    *cobra.Command
+		cmdFunc func(*testing.T, *cobra.Command)
+	}{
+
+		{
+			name: "get",
```

Comment:
Let's remove this test for the `-h` flag; for now we should focus on tests that interact with the API.